geoengine_partner/views/partner.xml

```diff
@@ -40,7 +40,7 @@
 
     <record id="geoengine_vector_layer_res_partner" model="geoengine.vector.layer">
```

Comment:
Looks like this should be moved to a data file instead of partner views.

Reply:
IMO as this is data to define the widget which is part of the view. It's fine to have it here. It could otherwise be moved in a dedicated file for geo config of the view but still in the `views` folder.